### PR TITLE
Added support for Paws 60 keyboard

### PR DESCRIPTION
**Pull request by HorrorTroll**
## Description

Added support for Paws 60 keyboard. A keyboard that inspired Bakeneko60, more details is in readme

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/horrortroll/paws60/config.h [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+#define DIODE_DIRECTION COL2ROW
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE 5
+
+/* Forcing to use NKRO instead 6KRO */
+#define FORCE_NKRO
+
+/* Change USB Polling Rate to 1000hz and a larger keys per scan for elite gaming */
+#define USB_POLLING_INTERVAL_MS 1
```
Comment (drashna) [inline keyboards/horrortroll/paws60/config.h:49]: Won't be needed for long. 
```suggestion
```

File: keyboards/horrortroll/paws60/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+
+enum layer_names {
+    _BASE = 0,
+    _FN = 1
```
Comment (drashna) [inline keyboards/horrortroll/paws60/keymaps/default/keymap.c:26]: No need to explicitly set a number here.  It starts at 0 and increments up by one, automatically. 
```suggestion
    _BASE,
    _FN,
```

File: keyboards/horrortroll/paws60/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+
+enum layer_names {
+    _BASE = 0,
+    _FN = 1
```
Comment (drashna) [inline keyboards/horrortroll/paws60/keymaps/via/keymap.c:26]:
```suggestion
    _BASE,
    _FN,
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/horrortroll/paws60/keymaps/via/keymap.c (hunk 3 above)
Comment (HorrorTroll) [reply keyboards/horrortroll/paws60/keymaps/via/keymap.c]: Done, thanks

**Review by drashna — APPROVED** [review]

File: keyboards/horrortroll/paws60/rules.mk [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+NKRO_ENABLE = yes           # Enable N-Key Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+AUDIO_ENABLE = no           # Audio output
```
Comment (fauxpark) [inline keyboards/horrortroll/paws60/rules.mk:18]:
```suggestion
AUDIO_ENABLE = no           # Audio output

LAYOUTS = 60_ansi
```

?

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/horrortroll/paws60/rules.mk (hunk 4 above)
Comment (HorrorTroll) [reply keyboards/horrortroll/paws60/rules.mk:18]: Oops, just added them now

**Review by fauxpark — APPROVED** [review]

**Pull request merged**